src/Microsoft.ML.Core/EntryPoints/ModuleArgs.cs

```diff
@@ -133,7 +133,8 @@ public sealed class OutputAttribute : Attribute
         /// A node can be run without optional input fields.
         /// </summary>
         [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
-        public sealed class OptionalInputAttribute : Attribute { }
+        [BestFriend]
+        internal sealed class OptionalInputAttribute : Attribute { }
```

Comment:
>OptionalInputAttribute [](start = 30, length = 22)

why only this get changed?
And it's a part of internal static class, isn't it internal already?